resources/aries-test-b/config.json

```diff
@@ -0,0 +1,275 @@
+{
+    "targetChainUrl": "wss://aries-relaynet-a.subspace.network/",
+    "primaryChain": {
+        "downloadedArchivePath": "/archives/kusama",
+        "wsUrls": [
+            "wss://kusama-rpc.polkadot.io",
+            "wss://kusama.api.onfinality.io/public-ws?apikey=38f60995-1ca8-474f-bbd1-bd80844433ad",
```

Comment:
Putting production secrets like API keys in the repo, especially public, is always a bad idea. I had to revoke these and @isSerge will have to redeploy all relayers now with new keys. These should be removed and replaced with obvious placeholders instead.